bundles/SystemInfoBundle/src/DependencyInjection/SystemInfoExtension.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Pimcore\Bundle\SystemInfoBundle\DependencyInjection;
+
+use Symfony\Component\Config\FileLocator;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Extension\Extension;
+use Symfony\Component\DependencyInjection\Loader\YamlFileLoader;
+
+class SystemInfoExtension extends Extension
```

Comment:
```suggestion
class PimcoreSystemInfoExtension extends Extension
```
also here, same as `bundles/SystemInfoBundle/src/SystemInfoBundle.php`